.github/workflows/static.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     steps:
       - uses: actions/checkout@master
       - name: Run PHPStan
-        uses: docker://jakzal/phpqa:php7.3-alpine
+        uses: docker://jakzal/phpqa:php8.1-alpine
```

Comment:
then you also need to make sure the tests run on php 8.1 as well: https://github.com/ETSGlobal/loco-adapter/blob/55caa8cf04c1000db45743889bc5d0d0e2c0713c/.github/workflows/ci.yml#L13